backend/include/sdk.hpp

```diff
@@ -53,6 +54,24 @@ class SDK {
     void handleAFVEventForWebsocket(sdk::types::Event event,
         const std::optional<std::string>& callsign, const std::optional<int>& frequencyHz);
 
+    /**
+     * @brief Builds a JSON object for the station state.
+     *
+     * @param callsign The callsign of the station. Optional.
+     * @param frequencyHz The frequency of the station.
+     *
+     * @return The JSON object for the station state.
+     */
+    nlohmann::json SDK::buildStationStateJson(
+        const std::optional<std::string>& callsign, const int& frequencyHz);
+
+    /**
+     * @brief Publishes the station state JSON to the websocket clients.
+     *
+     * @param state A JSON object representing the station state.
+     */
+    void SDK::publishStationState(const nlohmann::json& state);
```

Comment:
Exposed as its own method instead of using `handleAFVEventForWebsocket()` so the state object doesn't have to get created multiple times when sending via both websocket and IPC.